es-ds-docs/pages/atoms/layout.vue

```diff
@@ -1,5 +1,328 @@
 <template>
     <div>
-        atoms - layout
+        <h1>
+            Layout
+        </h1>
+        <p>
+            Extended from 
+            <NuxtLink to="https://bootstrap-vue.org/docs/components/layout" target="_blank">
+                bootstrap-vue layout
+            </NuxtLink>
+        </p>
+        <h2 class="mt-200">
+            Responsive Breakpoints
+        </h2>
+        <p>
+            At each breakpoint above extra small (xs), the content is constrained to the max width listed below.
+        </p>
+        
+        <ds-responsive-table>
```

Comment:
Yeah the responsive table does have a weakness of when some column headers wrap their text, the row height of the header column is inconsistent between columns.

<img width="769" alt="Screen Shot 2024-08-02 at 9 29 32 AM" src="https://github.com/user-attachments/assets/dad68a61-fca3-4a94-aa02-b1bae99543a7">

Not really in scope to refactor/fix responsive table for this project, so how about transposing the table, so the column headers become row headers, and the row headers become column headers? Then the tabler should be tall rather than wide and shouldn't run into this problem.

Reply:
When I transposed the table, I edited the table data constants to remove an empty row that was appearing. In `breakpointTableItems` and `breakpointTableFields` I commented out the label item and moved the labels to a new constant  `breakpointTableLabels.`